sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -2794,4 +2794,35 @@ class SubquerySuite extends QueryTest
       checkAnswer(df3, Row(7))
     }
   }
+
+  test("SPARK-49646: handle under/setOp under limit/aggregate with predicates cannot pulled up") {
+    sql("CREATE TABLE IF NOT EXISTS t(t1 INT,t2 int) USING json")
```

Comment:
My guess on the test failure: it's because the table `t` already exists (see [here](https://github.com/apache/spark/blob/931ab065df3952487028316ebd49c2895d947bf2/sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala#L67)) so this statement will do nothing.

Reply:
Thanks for catching that!